On some AMD platforms, the SMBus IO region 0xb00 - 0xb07 conflicts with the
corresponding ACPI SystemIO region. This prevents the SMBus host controller
driver to function correctly. We set acpi_enforce_resources to
ENFORCE_RESOURCES_LAX so that the SMBus driver can work normally, even
though a warning message notifying the conflict is printed in system logs.

Signed-off-by: Arindam Nath <arindam.nath@amd.com>
diff -Naur a/drivers/acpi/osl.c b/drivers/acpi/osl.c
--- a/drivers/acpi/osl.c	2013-11-04 05:11:51.000000000 +0530
+++ b/drivers/acpi/osl.c	2014-05-06 16:41:51.078524581 +0530
@@ -1533,7 +1533,7 @@
 #define ENFORCE_RESOURCES_LAX    1
 #define ENFORCE_RESOURCES_NO     0
 
-static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_STRICT;
+static unsigned int acpi_enforce_resources = ENFORCE_RESOURCES_LAX;
 
 static int __init acpi_enforce_resources_setup(char *str)
 {
